config/peerpods/mc-40-kata-remote-config.yaml

```diff
@@ -14,6 +14,6 @@ spec:
/rMl+X5mNpl84OusOObj0LErZqJBL9OnQL9Qrq3lngtA8xq2UIsGnItNKrEl8SLJ/ulDxZ/gGjFcb2WevLXh+n6cY4jvcIhXLegbqKEBb/c0HdWGpExmUM2OO0XnMTkZ+YB88wqdJV702GIEKqom9pmdM7kiS5Aa6WXbWtNaCrgi9FbaCGARVx8tMGLHnVQ+1VR7PYSRU9X5wuz0jFYPWdKYcepHxmGcUXW12lS+3jMRjh1GsjBh7/15PDNN07em9fdtQyWwMUUXMzeUQxrUBoN6cpBG9ZkMWfRpmOHt+qCXG1O01qzh5Nv01EDfqxGakw/vQyFQymAfoEbnMqpYinkYswpjLj9kcH52dgb0YAxd07qprPG+BgzsL786OzubpT/EixcvLs+zmcjUd+ffROe8bTfeX55lHyO+yjovdsYW1L+HlqKgHE3wx3lpvnIfL8nb4K8r7w7omB4h6fM3lO6TdOUNlRMFhn8ST8bbp2pfsptxnUje1NwS3BClnefodHg03ORrLKq/EkMcInVkOl5nwavf9BsLIPsnUUanzdAwZtaP0CH09ziBVw2VO6qFz/iOx/CiJ+1oC9axdZ4cHuuw5s+aBg/CIL4HGoU6XSjJa7SG3AdcXr1NBV+lObc8fEgjmvJsZFLgDnLxl2AZ5qr4PkuXayaExSpgsYpXd3p5e0KKJiCBpqtQkjuDwuM7obKmQ2e3G0n0V2enkynOGLzV89XZZPJBg2+M/nhEjcYMVWO08sbyIxT8vAsa9RGP4QSMy3hnxNfsBodyq0QRHTba+rAJU7InIotCcf5pMk0rhzbHeDfMU8u7ohp6GpNStLPQBj/GBPX2GkAnfH5DzzE57vnR+96LDoiEgVSR2C8myUU2rSfjOb7scZjF59nZIa8FoNFCDnxd3Tu7nKiqzeb32M8PgQWPHfTIwtRmMziMIBsJIcYB6ftHsCBjjXo0HF5Q6OECZ7gs6ERldtHF7APqwPiJE0KVprdskSHoPm0pKZnBz7oEw47avZH51ss2XEV/5wbsO9Ck90BFi8KV/NBvG4AEkIu0Rn/PZLBArAElscJ4u96nUEQKn1IdC/HSiC2QcVK2B+zzUtU+XaR/Ry+C5II/Cks2jRpXQ9N8yNiGsH0g0UMS4X+h3EkOZVt3G6XZa/aUhUo0ODgBvrU7UeMDXdGBXmZIBeLv2PUWHviCPDdNOz7w2JY/cFN4GGIWCkDUaDgsxIVsH3XzhodKdIEuVq36u94U17O3Gu9wpker4hrRHvFMatZLnc/RzPUcjnNP7z1xRpAuo874rICF+Mu0oKMXqwx7pOwtRbdUnLwFGGUhf5GwAdvP4mzkhp9Gmwd9fLr43a7c22Bly66uRWfrSOL/TuuJv93cvBY3lVUl2p+6ptz3Zx56ilorQ9wvlsva5LKujPMX518//vO3S9mqJe83m0x5T6MgJAs4sTuE9pd8ocGNHlWKtXQqF7Lz9CzXqI+H0V4k2GTB78FFbiIn7A/DTRAi7M+eO10mi3Xie9aLFnCV4vaLQXNdTP+7ww7ORu5FJbchkm+NDw/ppYckjNibznLSatCcQNdE0h3fmAvv3zDYm+6Rhf4Jg09999oOzaN2n+iiWa1wFfKhP42M56cvnHRccUQl94kI8cCgoHM/0b7wJ0fbCrL70CxUSZ+4jbJXtn0nrO5vLN0/Hs6S9E4g6Uc2Byhu5H++ev/2NF39HbzkJFNWqcAg3s6OkofOhffCm+DB9ATfHB0Gxn8k3JQF7LVRgjaQbfU5nqR2sXBvhlRl4rbo1BqNIl4ovmeSb6zp2ij6gzwYfXcjGqYLlOx4zcQvHbr3GgSSAqM0mkQPDfS5EgIccaSCGJWySwsQLvt4q3IKE4IqinCclz6lTMyWWnb4ncYD6NE1jOtLn172C8F4+OWg8JHC+NemeBtg9CWHB6CHCM9bGfuDOfNMRmcYwe9bOIA2KkJtTGFyqkB9pkEk0I3uBvefp8mxudm3MAkIrhjBFbJdqD/x61urOCCWSVbNphnewEM7GgPHWdzvaq10saISohMqvpxauHgD0FcuKFoaA4U4seYUvYvg1TIYcFw44Guiw2tMJB7c+aTKYcUWx4eG9c6rWv2Dgw7ovRoEdeJOB6/S6CJi0SMQLrkPqrtBhIO0sQ8Sl5iFzpWw495PfSCslV58OppY43WXwy8hPGTQn1KuZBjfw11rBr7QgEmNvcUg3PFlW0SZ33eJxOAb1n786ioaG7p2dOQ0R7FyQReJrKJelzo9Zlor52chE3shPmQym4lsnX2kdtwjE5iW6R/kOHC7D2gKm7lvzhQdRYmharvnrl0L8hbD5lZ6tVa18vt+r62FVtr0WMRabWISQKy7hjTX2/Qe0LGtUC9Wr2A/fDydDIcFYhzND6CibtFhFdSOxfqCTPr2cYqcY16A2CBE1otjOn3ap48QZC9+6SmpJ0FxxxtKgd/6dJgLMTZ5Ccw4oIc3RMUefJKxOHonXQi56ZnYtpY5VHyHRDx/JV6+uhHv3r4QVzf/ZfKBilUfR1kovrYQOIpjl1dlySk4DLLeXIUhjdRyQ4jE0ZGbf+yd9mOUCaNXJoAdpiTHWBSQXvG9hb1jIUBmCK7NE3L5g25RpTiEHNqPceiPOL/vyGAwQNdUqdXjhF59jacrWUPSo2CcR6OkkOThpL1O0YTxS8o4vXMdt5jGHEbssnf9IOk9uOjhJw1HzB2fR+bWdINH7EG8e/MTFwzq6PT2pEXiqALojiB1Q4KvTBE7L5Tmfjr8N71tEC6/7UO+fEiOYWthv78Zf2hAOn6QACU0bICrv8XFZJp2cFlILy+WIVhY3sJ+iVAGi0UFG59a+BytQ1omRb7oB9GS4oTIesoMEt7qlf1bmniYI7Tooe7lMl2aWSu9ZAzsUulliA8PKJJObMApi/7daKY/Q3FcdSNEERniEgy43uLpjVCJrkCzn9/Cfr625hbsojRm6UFL7Zd/4b/nqvh+8n8DAAD//8T7BEz6XAAA
```

Comment:
Could you add some explanation why this is changing?

Reply:
yes

Reply:
I added the original config toml file and added details about  how the machineconfig is created. Hope it helps. Let me know if you want me to add any additional info.